CHANGELOG.md

```diff
@@ -44,6 +44,14 @@ Documentation for rocRAND is available at
 
 ### Changes
 
+* For device-side generators, you can now wrap calls to rocrand_generate_* inside of a hipGraph. There are a few 
```

Comment:
I think we should consider adding an official documentation page in /docs on using hipGraphs with rocRAND (and also for the other Stream libraries, for that matter).  Essentially with the notes in the changelog entry. The information in the changelog is very useful, but would be very hard to find by an end-user in the future.

The deadline for 6.2 is getting pretty close so maybe we can do this for 6.3.

Reply:
I'll make sure we internally track documenting this (for rocPRIM as well, which AFAICT is the only other library we have been adding graph support for so far) and adding support for generating with a Poisson distribution.